content/docs/concepts/security/index.md

```diff
@@ -236,8 +219,50 @@ Istio provides two types of authentication:
       generation, distribution, rotation, and revocation.
 
 - Origin authentication, also known as end-user authentication: verifies the
-  original client making the request as an end-user or device. Istio
-  supports authentication with JSON Web Token (JWT) validation.
+  original client making the request as an end-user or device.
+  Istio enables request-level authentication with JSON Web Token (JWT) validation
+  and a streamlined developer experience for [Auth0](https://auth0.com/), [Firebase Auth](https://firebase.google.com/docs/auth/),
+  [Google Auth](https://developers.google.com/identity/protocols/OpenIDConnect), and custom auth.
+
+In both cases the authentication policies are stored in the `Istio config store` via a custom Kubernetes API,
+and are kept up to date (along with keys where appropriate) for each proxy by Pilot.
+Istio also supports authentication in permissive mode to understand how a policy change would affect your security posture
```

Comment:
Remove the conjunctive to achieve greater clarity:
Additionally, Istio supports authentication in permissive mode to help you understand how a policy change can affect your security posture before it becomes effective.